Gem/Code/Source/AutoGen/NetworkTestSpawnerComponent.AutoComponent.xml

```diff
@@ -12,6 +12,7 @@
 	<ComponentRelation Constraint="Required" HasController="false" Name="NetworkPrefabSpawnerComponent" Namespace="MultiplayerSample" Include="Source/Components/PerfTest/NetworkPrefabSpawnerComponent.h" />
 
 	<ArchetypeProperty Type="bool" Name="Enabled" Init="true" ExposeToEditor="true" Description="Enables spawning of test prefabs." />
+	<ArchetypeProperty Type="bool" Name="RespawnEnabled" Init="false" ExposeToEditor="true" Description="Deletes old instances and spawns new ones when at the maximum live count." />
```

Comment:
As maximum live count is a cvar, could we reference it by name? "when MaxiumumLiveCount is reached." Also is there anything folks should know about how instances are replaced? Are all replaced at once? Do we start replacing the oldest first? How does this function?